rust/crd/src/security.rs

```diff
@@ -190,6 +191,7 @@ impl DruidTlsSecurity {
         prepare: &mut ContainerBuilder,
         druid: &mut ContainerBuilder,
         pod: &mut PodBuilder,
+        requested_secret_lifetime: Duration,
```

Comment:
In all the other operators you passed it by reference ;) (remember to deref it in line 207)
```suggestion
        requested_secret_lifetime: &Duration,
```